forge-gui/res/cardsfolder/upcoming/queen_allenal_of_ruadach.txt

```diff
@@ -0,0 +1,12 @@
+Name:Queen Allenal of Ruadach
+ManaCost:G W W
+Types:Legendary Creature Elf Noble
+PT:*/*
+S:Mode$ Continuous | EffectZone$ All | CharacteristicDefining$ True | SetPower$ X | SetToughness$ X | Description$ CARDNAME's power and toughness are each equal to the number of creatures you control.
+R:Event$ CreateToken | ActiveZones$ Battlefield | ValidToken$ Card.YouCtrl | ReplaceWith$ DBReplace | Description$ If one or more tokens would be created under your control, those tokens plus a 1/1 white Soldier creature token are created instead.
+SVar:DBReplace:DB$ ReplaceToken | Type$ AddToken | Amount$ 1 | ValidCard$ Card.YouCtrl | TokenScript$ w_1_1_soldier
+SVar:X:Count$Valid Creature.YouCtrl
+SVar:BuffedBy:Creature
+SVar:NoZeroToughnessAI:True
+DeckHas:Ability$Token & Type$Soldier
```

Comment:
also add
DeckHints:Ability$Token (so the AI wants to put more card that make tokens in a deck with this)

Reply:
Added missing `DeckHints` flag